BenadrylAutomation/src/test/java/com/jnj/testcases/ProductPageTest.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.testcases;
+
+import org.testng.SkipException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.jnj.base.BaseClass;
+import com.jnj.dataproviders.ProductPageProvider;
+
+public class ProductPageTest extends BaseClass {
```

Comment:
@zahinSJI the format of the file doesn't look good. Pls format the file properly.

Reply:
Formatting updated

Reply:
fixed